doc/releases/release-notes-4.2.rst

```diff
@@ -222,6 +222,13 @@ New APIs and options
 
    * :kconfig:option:`CONFIG_NVME_PRP_PAGE_SIZE`
 
+* Sub-systems
```

Comment:
No need for a subsystem section, every top-level section is pretty much a subsystem already